services/kaze/controllers/application.go

```diff
@@ -228,6 +230,7 @@ func TransferApplicationOwnership(c *gin.Context) {
 
 // FetchMetrics retrieves the metrics of an application's container
 func FetchMetrics(c *gin.Context) {
```

Comment:
you don't need to change the functions in existing routes

Instead make a new route `/stream/:app/metrics` and put your function there